src/ssl.c

```diff
@@ -55025,6 +55054,61 @@ int wolfSSL_CONF_cmd(WOLFSSL_CONF_CTX* cctx, const char* cmd, const char* value)
         b->shutdown = BIO_CLOSE;
         return WOLFSSL_SUCCESS;
     }
+
+    int wolfSSL_BIO_do_accept(WOLFSSL_BIO *b)
+    {
+        SOCKET_T sfd = SOCKET_INVALID;
+        WOLFSSL_ENTER("wolfSSL_BIO_do_accept");
+
+        if (!b) {
+            WOLFSSL_ENTER("Bad parameter");
+            return WOLFSSL_FAILURE;
+        }
+
+        while (b && b->type != WOLFSSL_BIO_SOCKET)
```

Comment:
Accept only applies to SOCKET type? What about the SSL type, can that also do a wolfSSL_accept?

Reply:
>BIO_do_accept() serves two functions. When it is first called, after the accept BIO has been setup, it will attempt to create the accept socket and bind an address to it. Second and subsequent calls to BIO_do_accept() will await an incoming connection, or request a retry in non blocking mode. 

`BIO_do_accept` and `BIO_do_connect` abstract away the socket handling. This is different from accepting a TLS connection (although the naming is similar and prone to misunderstanding).